pkgs/development/libraries/qxmpp/default.nix

```diff
@@ -0,0 +1,40 @@
+{ mkDerivation
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, withGstreamer ? true
+, gst_all_1 ? null
+}:
+
+mkDerivation rec {
+  pname = "qxmpp";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "qxmpp-project";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1knpq1jkwk0lxdwczbmzf7qrjvlxba9yr40nbq9s5nqkcx6q1c3i";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ]
+  ++ lib.optionals withGstreamer (with gst_all_1; [
+    pkg-config
+    gstreamer
+    gst-plugins-bad
+    gst-plugins-base
+    gst-plugins-good
+  ]);
+  cmakeFlags = [ "-DWITH_GSTREAMER=ON" ];
+
+  meta = with lib; {
+    description = "Cross-platform C++ XMPP client and server library";
+    homepage = "https://github.com/qxmpp-project/qxmpp";
+    license = licenses.lgpl21;
```

Comment:
lgpl21Plus or lgpl21Only?

Reply:
Just 2.1: https://github.com/qxmpp-project/qxmpp/blob/master/LICENSE.LGPL

Reply:
Actually it is lgpl21Plus https://github.com/qxmpp-project/qxmpp/blob/master/src/client/QXmppArchiveManager.cpp#L15. The license texts for both are the same.